src/lib/routing/router.ts

```diff
@@ -20,21 +20,47 @@ const defaultErrorHandler = (auto405) => async (err: Error, req: EReq, res: ERes
   res.withStatus(500).json({ error: err.message });
 }
 
+const preflightHandler = (autoCorsPreflight: AutoCorsPreflightOptions) => async (req: EReq, res: ERes) => {
+  if(autoCorsPreflight.trustedOrigins.length===0){
+    return;
+  }
+  let originHeaderValue :string | null = null;
+  if(autoCorsPreflight.trustedOrigins.length===1 && autoCorsPreflight.trustedOrigins[0] === '*'){
+    originHeaderValue = '*';
+  }else{
+    const origin = req.headers.get("origin");
+    if (autoCorsPreflight.trustedOrigins.includes(origin)){
```

Comment:
Let's do this case insensitively, just in case I do something silly like `trustedOrigins: [ "ExAMplE.com" ]`